### ISSUE TITLE
No visible rows in when downgrade v2 to v1

### ISSUE THREAD

**Issue by cmonzon44** [onechiporenko/ember-models-table#262]
Well, i downgrade version to v1 to use templates in table but when all rows are not visible, all cells are in blank. To confirm my models are returning data to .hbs i put a foreach to print values and thats Ok.

if I don't change my code and upgrade to v2, Table prints ok (but column with template doesn't appear)

Any suggestions?

Thanks.

**Comment by onechiporenko** [comment]
@cmonzon44, create a demo pls on https://ember-twiddle.com/ and use issue-template on the github.

**Comment by cmonzon44** [comment]
Sorry but I can't re-create a the problem because on ember-twiddle it works because it use ember v 2.12 and the latest version is 2.15 (in my workspace).I create a demo on c9.io with 2.15  (it looks bad because i don't install bootstrap).

**Comment by cmonzon44** [comment]
code: https://ide.c9.io/carlosmmonzon/test-ember-models-table
preview: http://test-ember-models-table-carlosmmonzon.c9users.io:8080/tables

**Comment by steverhoades** [comment]
@cmonzon44 v1 will not work on 2.15.  This is due to partials being broken with that ember version, I believe that partials are still broken with 2.16 as well.

In order to get your template for your column to render you need to convert it from a partial to a component.  This should be as easy as placing it in `templates/components/template-name.hbs` and changing your column definition to the component name vs. the partial template path.

**Comment by cmonzon44** [comment]
Its different, in these case i haven't got any templates, only "data".

controller/tables.js
                 
```
import Ember from 'ember';

export default Ember.Controller.extend({
    
    columnas: [{
        propertyName: 'id',
        title: 'ID'
        },
        {
        propertyName: 'firstName',
        title: 'First Name'
        },
        {
        propertyName: 'lastName',
        title: 'Last Name'
        },
        {
        propertyName: 'city',
        title: 'City'
    }]
});

```

templates/tables.hbs

```
{{outlet}}

<h1>Title</h1>

{{models-table
    data=model
    columns=columnas
}}

{{#each model as |tab|}}
 <li>{{tab.firstName}} - {{tab.lastName}} </li>
{{/each}}
```

![image](https://user-images.githubusercontent.com/11430219/31642228-39e2825a-b2c0-11e7-9565-4bfc2dddd079.png)

**Comment by steverhoades** [comment]
@cmonzon44 It appears that they have fixed the partials issue with 2.15.  According to the thread they will be creating a release today - assuming this happens and you upgrade to the version with the fix for glimmer v1 should work.  Otherwise, if you are using 2.15 you must use v2.0 or your tables will not work.
